libraries/core/src/main/kotlin/com/bumble/appyx/core/navigation/transition/CombinedHandler.kt

```diff
@@ -24,5 +25,7 @@ class CombinedHandler<T, S>(
 }
 
 @Composable
-fun <T, S> rememberCombinedHandler(handlers: List<ModifierTransitionHandler<T, S>>): ModifierTransitionHandler<T, S> =
-    remember { CombinedHandler(handlers = handlers) }
+fun <T, S> rememberCombinedHandler(
```

Comment:
Let's deprecate the current `rememberCombinedHandler` and introduce a new one that takes `rememberCombinedHandler` in order not to break API and remove it in 1.1

Reply:
Good idea. Will do